docs/features/tenancy.md

```diff
@@ -12,28 +13,112 @@ Key attributes of tenancy:
-limits at granularity of datastore.
+- Vmgroups can share the same underlying storage but preserve volume namespace isolation limits at granularity of datastore.
 
 ## Admin CLI
 
 Vmgroups can be created and managed via the [Admin CLI](/user-guide/admin-cli/#Vmgroup)
 
-## Default vmgroup
-When a VM which does not belong to any vmgroup issues a request to vmdk_ops, this VM will be assumed to be in _DEFAULT vmgroup, and will get privileges
-associated with this vmgroup. \_DEFAULT vmgroup will be automatically created by system post install, so by default vmdk_ops will support request from
-any VM , thus maintaining backward compatibility and simplicity of installation.An admin can remove this vmgroup or modify privileges, thus locking
-down vmdk_ops to serve only explicitly configured VMs.
+## Multitenancy concepts
 
-## Default privileges
-When a VM tries to manipulate volumes on a datastore which is not configured,  the system will use _DEFAULT privilege, which is created automatically
-by system post install. This _DEFAULT privilege allows access to ANY datastore by default. An admin can edit or remove this record, thus locking down
-the functionality to allow access only to explicitly configured datastores.
+### Default vmgroup
 
-## Default datastore
+When a VM which does not belong to any vmgroup issues a request to vmdk_ops, this VM will be assumed to be in _DEFAULT vmgroup, and will get privileges associated with this vmgroup. \_DEFAULT vmgroup will be automatically created by system post install, so by default vmdk_ops will support request from any VM , thus maintaining backward compatibility and simplicity of installation.An admin can remove this vmgroup or modify privileges, thus locking down vmdk_ops to serve only explicitly configured VMs.
+
+```
+#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
+Uuid                                  Name      Description                Default_datastore  VM_list
+------------------------------------  --------  -------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vmgroup
+```
+
+### Default privileges
+When a VM tries to manipulate volumes on a datastore which is not configured, the system will use _DEFAULT privilege, which is created automatically by system post install. This _DEFAULT privilege allows access to ANY datastore by default. An admin can edit or remove this record, thus locking down the functionality to allow access only to explicitly configured datastores.
+
+### Default datastore
 When a VM addresses the volume using short notation (volume_name, without @datastore), all VMs in this vmgroup will use default datastore to resolve short volume reference (volume_name will actually mean volume_name@default_datastore).
+
 If "default_datastore" is not set for a vmgroup, then datastore where the VM resides will be used as "default_datastore".
 
+## Example
+
```

Comment:
The following example need to be updated. "default_datastore" must be specified when creating a vmgroup.